nf_core/lint.py

```diff
@@ -68,19 +68,41 @@ def run_linting(pipeline_dir, release_mode=False):
     if len(lint_obj.failed) > 0:
         logging.error(
             "Sorry, some tests failed - exiting with a non-zero error code...{}\n\n"
-            .format("\n       Reminder: Lint tests were run in --release mode." if release else '')
+            .format("\n\tReminder: Lint tests were run in --release mode." if release_mode else '')
         )
 
     return lint_obj
 
 
 class PipelineLint(object):
-    """ Object to hold linting info and results """
-
-    def __init__(self, pipeline_dir):
+    """Object to hold linting information and results.
```

Comment:
@ewels this is an example (not complete) docstring in Python style format. The other docstring format convenvtion I found is from NumPy.

Reply:
of course if there is some convention based on Markdown, from which API docs can be generated, I am happy to give it a try.